GenericsHomework/GenericsHomework.Tests/NodeTests.cs

```diff
@@ -0,0 +1,65 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace GenericsHomework.Tests
+{
+    [TestClass]
+    public class NodeTests
+    {
+        private (Node<int>,Node<string>) makeNodes()
+        {
+            return (new Node<int>(10), new Node<string>("Miracle Max"));
+        }
+
+        [TestMethod]
+        public void Node_ConstructorIsGeneric()
+        {
+            var (intNode, stringNode) = makeNodes();
+        }
+
+        [TestMethod]
+        public void Node_ToStringWorks()
+        {
+            var (intNode, stringNode) = makeNodes();
+
+            Assert.AreEqual<string>(intNode.ToString(), "10");
+            Assert.AreEqual<string>(stringNode.ToString(), "Miracle Max");

```

Comment:
```suggestion
            Assert.AreEqual(intNode.ToString(), "10");
            Assert.AreEqual(stringNode.ToString(), "Miracle Max");
```
Not needed. This is true for all of the `Asserts` that do this.